oauth2client/contrib/django_util/decorators.py

```diff
@@ -87,20 +110,22 @@ def optional_oauth2(request):
                # this could be passed into a view
                # request.oauth.http is also initialized
                return HttpResponse("User email: {0}".format(
-                                   request.oauth.credentials.id_token['email']))
+                                   request.oauth.credentials.id_token['email'
            else:
                return HttpResponse('Here is an OAuth Authorize link:
                <a href="{0}">Authorize</a>'.format(
                    request.oauth.get_authorize_redirect()))
 
 
-    :param decorated_function: View function to decorate
-    :param scopes: Scopes to require, will default
-    :param decorator_kwargs: Can include ``return_url`` to specify the URL to
+    Args:
+        decorated_function: View function to decorate
```

Comment:
Missing trailing punctuation on parameter and return value descriptions, here and below.